modules/talos/manifests/init.pp

```diff
@@ -11,7 +11,7 @@
             include httpd
             include packages::java_developer_package_for_osx
             include packages::xcode_cmd_line_tools
-            require dirs::builds
+            #require dirs::builds
```

Comment:
Why is this commented out?

Reply:
same as the other, puppet tries to change permission on the synthetic.conf hard links